--- ../src-base/minecraft/net/minecraft/command/CommandClone.java
+++ ../src-work/minecraft/net/minecraft/command/CommandClone.java
@@ -51,9 +51,9 @@
             StructureBoundingBox structureboundingbox1 = new StructureBoundingBox(blockpos2, blockpos2.func_177971_a(structureboundingbox.func_175896_b()));
             int i = structureboundingbox.func_78883_b() * structureboundingbox.func_78882_c() * structureboundingbox.func_78880_d();
 
-            if (i > 32768)
+            if (i > net.minecraft.gnembon.CarpetSettings.fillCommandLimit)
             {
-                throw new CommandException("commands.clone.tooManyBlocks", new Object[] {i, Integer.valueOf(32768)});
+                throw new CommandException("commands.clone.tooManyBlocks", new Object[] {i, Integer.valueOf(net.minecraft.gnembon.CarpetSettings.fillCommandLimit)});
             }
             else
             {
