--- ../src-base/minecraft/net/minecraft/command/CommandFill.java
+++ ../src-work/minecraft/net/minecraft/command/CommandFill.java
@@ -60,9 +60,9 @@
             BlockPos blockpos3 = new BlockPos(Math.max(blockpos.func_177958_n(), blockpos1.func_177958_n()), Math.max(blockpos.func_177956_o(), blockpos1.func_177956_o()), Math.max(blockpos.func_177952_p(), blockpos1.func_177952_p()));
             int i = (blockpos3.func_177958_n() - blockpos2.func_177958_n() + 1) * (blockpos3.func_177956_o() - blockpos2.func_177956_o() + 1) * (blockpos3.func_177952_p() - blockpos2.func_177952_p() + 1);
 
-            if (i > 32768)
+            if (i > net.minecraft.gnembon.CarpetSettings.fillCommandLimit)
             {
-                throw new CommandException("commands.fill.tooManyBlocks", new Object[] {i, Integer.valueOf(32768)});
+                throw new CommandException("commands.fill.tooManyBlocks", new Object[] {i, Integer.valueOf(net.minecraft.gnembon.CarpetSettings.fillCommandLimit)});
             }
             else if (blockpos2.func_177956_o() >= 0 && blockpos3.func_177956_o() < 256)
             {
